packages/api/cms-api/package.json

```diff
@@ -121,7 +114,6 @@
     },
     "peerDependencies": {
         "@kubernetes/client-node": ">=0.18.0",
-        "@mikro-orm/cli": "^6.0.0",
```

Comment:
We can remove the respective changeset and info in the migration guide then.

Reply:
Removed the changeset and updated migration guide

Reply:
will add another section into the migration guide, that @comet/api-generator is available.

Reply:
done